### ISSUE TITLE
About the accuracy of supervised learning with labeled data

### ISSUE THREAD

**Issue by adfhuydx** [OatmealLiu/class-iNCD#9]
Hi, I am very interested in your work. 
Could you please tell me **the accuracy of supervised learning with labeled data** on the three datasets (5  base classes for cifar10, 80  base classes for cifar100, and 180  base classes for tinyimagenet) ? I see that this value is not reported in the paper. 
Thank you very much.

**Comment by OatmealLiu** [comment]
Hi, thanks for your interests in our work.

The numbers reported in the paper for the base classes (Old) are the final results achieved by the model after incremental discovery step. The results were obtained by testing on base class only with joint head. During the base class supervised pre-training stage, base class labels are used. Therefore, that results are actually supervised results after incremental discovery step (of course, it forgets the base class knowledge to a certain degree).

Thanks!

Best regards,
Miu
